galgebra/utils.py

```diff
@@ -9,12 +9,8 @@
 PY2 = sys.version_info[0] == 2
 PY3 = sys.version_info[0] == 3
```

Comment:
Looks like these lines (and `import sys` above) also should be removed — or am I missing something?

Reply:
This module is public API, so needs to stick around probably for another release. Imagine a project that works in both 2 and 3, and depends on galgebra.utils. Such a project should keep working in 3 after this change, even though `utils` isn't doing anything useful any more.

Reply:
We could actually drop `utils` in entirety in a later PR since I'm pretty sure no one would have been using in any serious way after I introduced it.